android/build.gradle

```diff
@@ -1,22 +1,31 @@
 buildscript {
     repositories {
         jcenter()
+        google()
```

Comment:
I recommend replacing `google()` with:
```
maven {
    url 'https://maven.google.com/'
    name 'Google'
}
```

Reply:
agreed

Reply:
@rayronvictor we should also swap google and jcenter due to recent issues with resolutions from jcenter